packages/components/virtual-list/src/builders/buildGrid.ts

```diff
@@ -35,6 +37,10 @@ import type { CSSProperties, Slot, VNode, VNodeChild } from 'vue'
 import type { GridConstructorProps, Alignment } from '../types'
 import type { VirtualizedGridProps } from '../props'
 
+type ScrollbarExpose = {
```

Comment:
Should we move this to `scroll-bar` and export it?

Reply:
Right